packages/hooks/package.json

```diff
@@ -24,12 +24,17 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"exports": {
+		".": {
+			"types": "./build-types/index.d.ts",
+			"import": "./build-module/index.js",
+			"require": "./build/index.js"
+		},
+		"./package.json": "./package.json"
+	},
```

Comment:
The addition of the exports is not necessary, but I'm using this as an excuse to also migrate our packages to the recent guidelines about shipping npm packages. "main" and "module" are deprecated and could be removed but I kept them for old tools for now. We may consider removing them later.

Reply:
Yes, there is a webpack config currently that uses it and we can use findPackageJSON because the node.js version is not supported yet and I didn't want to upgrade node in this PR.